src/torchcodec/samplers/_implem.py

```diff
@@ -140,3 +140,61 @@ def clips_at_random_indices(
     random.setstate(builtin_random_state)
 
     return clips
+
+
+def clips_at_regular_indices(
+    decoder: VideoDecoder,
+    *,
+    num_clips: int = 1,
```

Comment:
I feel like `clips_at_regular_indices(num_clips=1)` is an anti pattern. Same with 2. Maybe we should make 3 the default, and enforce `num_clips>=3`?

I don't feel too strongly about it either way - just being pedantic.

Reply:
If this is what other libraries do we can do the same here.

I was thinking of sampling the middle of the video if num_clips=1 but that would make it slower than existing libraries

Reply:
OK. Other libs allow `num_clips=1`. Let's revisit if needed